src/alchemlyb/estimators/bar_.py

```diff
@@ -128,5 +128,7 @@ def fit(self, u_nk):
         self.d_delta_f_ = pd.DataFrame(np.sqrt(ad_delta + ad_delta.T),
                                        columns=self.states_,
                                        index=self.states_)
+        self.delta_f_.attrs = u_nk.attrs
+        self.d_delta_f_.attrs = u_nk.attrs
```

Comment:
Does this raise a warning or is this find with pandas ≥ 1.0 ?

Reply:
@orbeckst There isn't any warning as https://travis-ci.org/github/alchemistry/alchemlyb/jobs/773410900 shows the py36 did not give more warning than py37.